docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -86,6 +86,38 @@ The `-e` option is to ensure the official Azure CLI package is installed. This c
 
 You can now run the Azure CLI with the `az` command from either Windows Command Prompt or PowerShell.
 
+## Enable Tab Completion on PowerShell
+
+When typing in a command followed by a parameter, pressing the tab key will display a list of available options for that parameter. This saves you time and effort in remembering specific arguments or looking them up in the reference documentation.  This capability applies to commands and parameters and also extends to values such as account name and subscription name.
+
+To enable tab completion for the Azure CLI on PowerShell, you need to have version 2.49 or higher installed.
+
+To add tab completion, create or edit the profile stored in the variable `$PROFILE`. The simplest way is to run `notepad $PROFILE` in PowerShell. For more information, see [How to create your profile](/powershell/module/microsoft.powershell.core/about/about_profiles#how-to-create-a-profile) and [Profiles and execution policy](/powershell/module/microsoft.powershell.core/about/about_profiles#profiles-and-execution-policy).
+
+Add the following code to your profile:
+
+```powershell
+Register-ArgumentCompleter -Native -CommandName az -ScriptBlock {
+    param($commandName, $wordToComplete, $cursorPosition)
+    $completion_file = New-TemporaryFile
+    $env:ARGCOMPLETE_USE_TEMPFILES = 1
+    $env:_ARGCOMPLETE_STDOUT_FILENAME = $completion_file
+    $env:COMP_LINE = $wordToComplete
+    $env:COMP_POINT = $cursorPosition
+    $env:_ARGCOMPLETE = 1
+    $env:_ARGCOMPLETE_SUPPRESS_SPACE = 0
+    $env:_ARGCOMPLETE_IFS = "`n"
+    az 2>&1 | Out-Null
+
+    Get-Content $completion_file | Sort-Object | ForEach-Object {
+        [System.Management.Automation.CompletionResult]::new($_, $_, "ParameterValue", $_)
+    }
+    Remove-Item $completion_file, Env:\_ARGCOMPLETE_STDOUT_FILENAME, Env:\ARGCOMPLETE_USE_TEMPFILES, Env:\COMP_LINE, Env:\COMP_POINT, Env:\_ARGCOMPLETE, Env:\_ARGCOMPLETE_SUPPRESS_SPACE, Env:\_ARGCOMPLETE_IFS
+}
+```
+
+To get a bash like completion style, add `Set-PSReadlineKeyHandler -Key Tab -Function MenuComplete` in `$PROFILE`.
```

Comment:
```suggestion
To get a Bash-like completion style, add `Set-PSReadlineKeyHandler -Key Tab -Function MenuComplete` in `$PROFILE`.
```

Not exactly Bash-like. Bash doesn't allow selecting with arrow keys:

![image](https://github.com/MicrosoftDocs/azure-docs-cli/assets/4003950/ed443e85-b839-4cbf-ad61-4b940f59d545)

but PowerShell does:

![image](https://github.com/MicrosoftDocs/azure-docs-cli/assets/4003950/1dd8dd80-f9a6-45eb-9d13-282378f83e19)

Reply:
How about changing to "To display all available options in the menu"?

PS: The option sequence is different from mine.
![image](https://github.com/MicrosoftDocs/azure-docs-cli/assets/2227874/45a31360-aa29-4380-a275-acdaddaeec77)